recipes/backward-cpp/all/conanfile.py

```diff
@@ -36,16 +36,27 @@ def _has_stack_walking(self, type):
     def _has_stack_details(self, type):
         return self.options.stack_details == type
     
+    def _supported_os(self):
+        return ["Linux", "Macos", "Android", "Windows"] if tools.Version(self.version) >= "1.5" \
+               else ["Linux", "Macos", "Android"]
+    
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+    
     def configure(self):
-        if self.settings.os not in ["Linux", "Macos", "Android"]:
+        if self.settings.os not in self._supported_os():
             raise ConanInvalidConfiguration("upstream backward-cpp v{0} is not \
                 supported in {1}.".format(self.version, self.settings.os))
-        # windows implementation only available in upstream master branch
 
         if self.settings.os == "Macos" and \
            not self._has_stack_details("backtrace_symbol"):
             raise ConanInvalidConfiguration("only stack_details=backtrace_symbol"
                                             " is supported on Macos")
+        if self.settings.os == "Windows" and \
+            not self._has_stack_details("pdb"):
```

Comment:
If I understand, we won't provide packages for Windows by default, is that right? 

@danimtb Could we manage the default_option by settings? e.g set "pdb" when building for Windows.

Reply:
Yes, there will be no packages for windows in this case. Generating packages for custom options is currently not possible but we may consider this for the future. However, I still think that even if you have to ``conan install --build``, the gain and value of using Conan and conan-center-index recipes is still ahead of using any other tool/approach

Reply:
What about enforcing this option on Windows? 🤔

Reply:
Well, as it's only option allowed on Windows, in this case I see no problem. So we can remove stack_details option for Windows and force pdb.

Reply:
if pdb is the "default" option used for windows, I am fine with it

Reply:
after rechecking the recipe, I totally agree with @uilianries. Multiple values for this option in windows has no sense according to the logic of the recipe

Reply:
Yep, when I added `"pdb"` to the options list I wasn't totally sure. The thing is that the library forces the value of `BACKWARD_HAS_PDB_SYMBOL` on windows (https://github.com/bombela/backward-cpp/blob/v1.5/backward.hpp#L342). 
I'll remove it now.